Solutions/Marain.UserNotifications.Specs/Features/ManagementApiClient/NotificationTemplate.feature

```diff
@@ -49,13 +49,18 @@ Scenario: Get a Web Push notification template
 		"""
 	When I use the client to send the notification template API a request to get a notification template with notification type 'marain.test.template4' and communication type 'WebPush'
 	Then the client response status code should be 'OK'
+	And the client response should contain a 'If-None-Match' header
```

Comment:
I don't quite understand this. `If-None-Match` is a *request* header (by definition) - you'd never expect to see it in a response.

If an etag is being returned in a response, it should be in the `Etag` header.

Reply:
Okay. Sorry, I might have got confused while reading that from the host response and passed that straight through. I shall fix that. Thanks @jongeorge1

Reply:
@jongeorge1 I have updated it to add the ETag header correctly. Thanks for that 😊